app/server/api/organisation_api.py

```diff
@@ -62,6 +62,8 @@ def post(self, organisation_id):
         token_id = post_data.get('token_id')
         deploy_cic = post_data.get('deploy_cic', False)
 
+        initial_disbursement_amount = post_data.get('initial_disbursement_amount')
```

Comment:
@PhilipWafula @michielderoos, if we are updating `initial_disbursement_amount` in this PR then we should be doing it across the whole stack and deprecating `DEFAULT_INITIAL_DISBURSEMENT` in the configs.

In my opinion, we don't want to be running dual disbursement amounts. This should not be merged in until resolved.
a) deprecate old config
b) rollback on this branch and split out to a separate PR

Reply:
Let me know what decision y'all decide. I started working on it here also without realising it was part of this PR. https://github.com/teamsempo/SempoBlockchain/pull/175

Can turn that PR into the ability to update from admin dashboard

Reply:
I've closed my PR in favor of this. Can we get this merged in soon?

Reply:
@tristanhcole we spoke with @michielderoos and agreed we should split the organization initial disbursement value out into its own PR so we're gonna scale back that feature from this PR and make sure it is on its own.
I think that's a good idea because Will requested that he get auto trickle bonus and auto intial bonus features out first before we get the ussd self service out.
So I have been focused on getting those done.
What I will do is split this initial bonus thing out, then deprecate the config one and get that as a stand alone feature merged.
Then the self service PR can follow once the auto trickle bonus ones are up